src/deps.deno.ts

```diff
@@ -0,0 +1 @@
+export { Bot, Context, NextFunction, MiddlewareFn } from 'https://deno.land/x/grammy@1.6.1/mod.ts'
```

Comment:
This would need updating every patch, what's the accepted practice @KnorpelSenf?

Reply:
> This would need updating every patch, what's the accepted practice @KnorpelSenf?

I'm using some tool in my repositories for auto updating dependicices, which can be used here too: https://github.com/grammyjs/storages/blob/main/.github/workflows/deno_deps_update.yml

Reply:
I'm not sure about deno practices, but in node i'm prefer to bind versions,

Reply:
That works. Otherwise you can replace deno.land by lib.deno.dev. Then you can specify semantic versions in the URLs. That's what grammY plugins do, cause I cannot be bothered to update 20 repos after every patch release. Unfortunately, Deno often doesn't compile if there are tiny dependency mismatches because you're effectively downloading grammY twice, so some internal things don't match anymore.